### PR TITLE
[BGP docker]: start bgp_eoiu_mark service to populate bgp eoiu marker…

### PR DESCRIPTION
**Pull request by jipanyang**
… flags for warm start

Signed-off-by: Jipan Yang <jipan.yang@alibaba-inc.com>

**- What I did**

Three PRs for adding BGP eoiu support to speed up route reconciliation in fpmsyncd

sonic-buildimage:  https://github.com/Azure/sonic-buildimage/pull/2823
sonic-swss-common: https://github.com/Azure/sonic-swss-common/pull/273
sonic-swss:  https://github.com/Azure/sonic-swss/pull/856

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: dockers/docker-fpm-quagga/start.sh [hunk 1]
```diff
@@ -18,6 +18,7 @@ echo "# Config files managed by sonic-config-engine" > /var/sonic/config_status
 
 rm -f /var/run/rsyslogd.pid
 
+supervisorctl start bgp_eoiu_marker
```
Comment (nikos-github) [inline dockers/docker-fpm-quagga/start.sh]: This needs to be configurable and not start by default.

Comment (jipanyang) [reply dockers/docker-fpm-quagga/start.sh]: thanks, I plan to drop the support for quagga, and enable it for frr only.

Comment (nikos-github) [reply dockers/docker-fpm-quagga/start.sh]: Even for frr, this need to be configurable. We can't have this starting by default and doing cli scraping every second.

Comment (jipanyang) [reply dockers/docker-fpm-quagga/start.sh]: Not sure if you checked the script.  
It exit immediately if it is cold start.  For warm start, it exits itself after collecting eor for all neighbors or after 120 seconds.

Comment (nikos-github) [reply dockers/docker-fpm-quagga/start.sh]: Even for warm start this should be configurable.

**Comment by nikos-github** [comment]
@jipanyang Have you addressed my comment?

**Comment by pavel-shirshov** [comment]
closing it. we already have https://github.com/Azure/sonic-buildimage/pull/3489

**Pull request closed without merging**